cmd/bucket-handlers.go

```diff
@@ -973,7 +973,7 @@ func (api objectAPIHandlers) PostPolicyBucketHandler(w http.ResponseWriter, r *h
 		if fileName == "" {
 			// value, store as string in memory
 			n, err := io.CopyN(&b, part, maxMemoryBytes+1)
-			part.Close()
+			defer part.Close()
```

Comment:
this is wrong @jiuker there is a `continue` here part is replaced with NextPart()